syntax = "proto2";

package simConnect;

message simData {
    required string  szTitle = 1;
    required double  dAbsoluteTime = 2;
    required double  dTime = 3;
    required double  usimOnGround = 4;
    required double  dAltitude = 5;
    required double  dHeading = 6;
    required double  dSpeed = 7;
    required double  dVerticalSpeed = 8;
    required double  dGpsEta = 9;
    required double  dLatitude = 10;
    required double  dLongitude = 11;
    required double  dSimTime = 12;
    required double  dTemperature = 13;
    required double  dAirPressure = 14;
    required double  dWindVelocity = 15;
    required double  dWindDirection = 16;
}
